test/common/lib/math-256.test.sol

```diff
@@ -0,0 +1,240 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "forge-std/Test.sol";
+import { Math256 } from "contracts/common/lib/Math256.sol";
+
+contract Math256Test is Test {
+
+    /// int256 tests for max/min
+
+    function testMaxUint256Simple1() public {
+        uint256 a = 1;
+        uint256 b = 2;
+
+        assertEq(Math256.max(a, b), b);
+    }
+
+    function testMaxUint256Simple2() public {
+        uint256 a = 1;
+        uint256 b = 2;
+
+        assertEq(Math256.max(b, a), b);
+    }
+
+    function testMaxUint256Simple3() public {
+        uint256 a = 1;
+        uint256 b = 1;
+
+        assertEq(Math256.max(b, a), b);
+    }
+
+    function testMaxUint256Fuzz(uint256 a, uint256 b) public {
+        uint256 expected;
+        
+        if (a > b) {
+            expected = a;
+        } else if (a == b) {
```

Comment:
can exclude this branch (when `a == b`)

Reply:
the same is true for other similar cases

Reply:
Fixed these cases.